frontend/app/embed.ts

```diff
@@ -329,9 +326,22 @@ function createInstance(config: CommentsConfig) {
     window.removeEventListener('message', receiveMessages);
     window.removeEventListener('hashchange', postHashToIframe);
     document.removeEventListener('click', postClickOutsideToIframe);
+
+    if (titleObserver) {
+      titleObserver.disconnect();
+    }
+
     iframe.remove();
   }
 
+  // TODO: These do not appear in Chrome DevTools
```

Comment:
I have this unresolved problem. Not really a problem to me, but may affect backward compatibility...

Reply:
As I see. API appears  in DevTools
<img width="1206" alt="Screenshot 2020-07-11 at 01 14 37" src="https://user-images.githubusercontent.com/2330682/87207230-1fe3b380-c314-11ea-858e-3523a9a388a2.png">